qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -91,6 +97,8 @@ def change_virtio_media(cdrom_virtio):
         win_driver_installer_test.install_gagent(session, test, qemu_ga_pkg,
                                                  gagent_install_cmd,
                                                  gagent_pkg_info_cmd)
+    error_context.context("Run viofs service...")
+    win_driver_installer_test.run_viofs_service(test, params, session)
 
     error_context.context("Upgrade virtio driver to original",
```

Comment:
After upgrade driver to original one, we also need to start virtiofs service.
    error_context.context("Run viofs service...")
    win_driver_installer_test.run_viofs_service(test, params, session)